src/main/java/seedu/ddd/logic/commands/AddEventCommand.java

```diff
@@ -0,0 +1,128 @@
+package seedu.ddd.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.ddd.logic.parser.CliSyntax.PREFIX_CLIENTS;
+import static seedu.ddd.logic.parser.CliSyntax.PREFIX_DESC;
+import static seedu.ddd.logic.parser.CliSyntax.PREFIX_VENDORS;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import seedu.ddd.commons.util.ToStringBuilder;
+import seedu.ddd.logic.Messages;
+import seedu.ddd.logic.commands.exceptions.CommandException;
+import seedu.ddd.model.AddressBook;
+import seedu.ddd.model.Model;
+import seedu.ddd.model.contact.client.Client;
+import seedu.ddd.model.contact.common.Id;
+import seedu.ddd.model.contact.vendor.Vendor;
+import seedu.ddd.model.event.common.Description;
+import seedu.ddd.model.event.common.Event;
+import seedu.ddd.model.event.common.EventId;
+
+/**
+ * Adds an event to DDD.
+ */
+public class AddEventCommand extends Command {
+
+    public static final String COMMAND_WORD = "event";
```

Comment:
I think it makes more sense if the command word is add. For listing, the `ListCommandParser` processes the following

- `list` lists all contacts
- `list -c` lists all clients
- `list -v` lists all vendors
- `list -e` lists all events

So instead of `event d/DESCRIPTION ...`, I think `add -e d/DESCRIPTION ...` would be more consistent.

Reply:
I feel that lumping the command together with add seems to violate the single responsibility principle. I prefer the current approach of separating the two commands and their implementations separate as they are don't really share similar business logic.